0.7.0 (#5)
MetaMask/snap-template: CHANGELOG.md

## [Unreleased]

## [0.7.0]

> ### Uncategorized
> - @metamask/snaps-cli@0.7.0 ([#4](https://github.com/MetaMask/snap-template/pull/4))
> - Rebuild snap
> - Fix lockfile
> - Add more content to the custom confirmation
> - Remove allow-scripts step from CI
> - Add cleanup script, more instructions in readme
> - Various template repository preparations
> - The beginning of time

Reviewer: ```suggestion
### Changed
- @metamask/snaps-cli@0.7.0 ([#4](https://github.com/MetaMask/snap-template/pull/4))
- Add more content to the custom confirmation
- Add cleanup script, more instructions in readme
```
